### PR TITLE
feat: iwd activity feed added

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- IWD activity feed components added

Screenshot 2024-02-02 at 9 40 52
Screenshot 2024-02-02 at 9 41 04

### CODE REVIEW

File: src/components/Iwd/ActivityAvatar.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<template>
+	<div class="tw-w-4 tw-h-4" v-if="!isDefaultProfilePic(imageFilename) && !isAnonymousUser && imageFilename">
+		<img
+			:src="lenderImageUrl"
+			alt="Image of lender"
+			class="tw-rounded-full tw-inline-block tw-w-4 tw-h-4"
```
Comment (dyersituations) [inline src/components/Iwd/ActivityAvatar.vue:6]: data-hj-suppress

File: src/components/Iwd/ActivityFeed.vue [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	inject: ['apollo'],
+	data() {
+		return {
+			activities: [],
+		};
+	},
+	methods: {
+		fetchActivities() {
+			// Fetch activities from API
+			this.apollo.query({
```
Comment (dyersituations) [inline src/components/Iwd/ActivityFeed.vue:29]: Do you think we need a loader?

File: src/components/Iwd/ActivityFeed.vue [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		},
+	},
+	mounted() {
+		this.fetchActivities();
+	},
+};
+</script>
+
+<style scoped lang="postcss">
+.hide-scrollbar {
```
Comment (dyersituations) [inline src/components/Iwd/ActivityFeed.vue:43]: Testing locally this means you can't really scroll on desktop. Once this is on dev, could you tag Nathan in our team channel about how he wants to handle this for desktop?

File: src/components/Iwd/ActivityFeed.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/components/Iwd/ActivityFeed.vue:29]: I considered this but should be good to have dynamic data to test this to see how is working. Locally is looking fine, but a good call to test in dev.
An edge case is returning non-data, so, no sure what is better, showing a loader and then removing this or not showing a loader and then show the feed if having data...

File: src/components/Iwd/ActivityFeed.vue (hunk 3 above)
Comment (roger-in-kiva) [reply src/components/Iwd/ActivityFeed.vue:43]: Yeah, lets see what is happening with dynamic data and what limit we gonna consider for this...

Comment (roger-in-kiva) [reply src/components/Iwd/ActivityFeed.vue:43]: A good desktop UX is having arrows to navigate on it, but lets see what is happening then.

**Review by dyersituations — APPROVED** [review]
